internal/services/netapp/netapp_account_resource.go

```diff
@@ -102,8 +102,58 @@ func resourceNetAppAccount() *pluginsdk.Resource {
 							ValidateFunc: validation.StringIsNotEmpty,
 						},
 						"organizational_unit": {
-							Type:     pluginsdk.TypeString,
-							Optional: true,
+							Type:        pluginsdk.TypeString,
+							Optional:    true,
+							Default:     "CN=Computers",
+							Description: "The Organizational Unit (OU) within the Windows Active Directory where machines will be created. If blank, defaults to 'CN=Computers'",
+						},
+						"site_name": {
+							Type:        pluginsdk.TypeString,
+							Optional:    true,
+							Default:     "Default-First-Site-Name",
+							Description: "The Active Directory site the service will limit Domain Controller discovery to. If blank, defaults to 'Default-First-Site-Name'",
+						},
+						"ad_name": {
+							Type:        pluginsdk.TypeString,
+							Optional:    true,
+							Description: "Name of the active directory machine. This optional parameter is used only while creating kerberos volume.",
+						},
+						"kdc_ip": {
+							Type:         pluginsdk.TypeString,
+							Optional:     true,
+							ValidateFunc: validation.IsIPv4Address,
+							Description:  "IP address of the KDC server (usually same the DC). This optional parameter is used only while creating kerberos volume.",
+						},
+						"enable_aes_encryption": {
+							Type:        pluginsdk.TypeBool,
+							Optional:    true,
+							Default:     false,
+							Description: "If enabled, AES encryption will be enabled for SMB communication.",
+						},
+						"allow_local_nfs_users_with_ldap": {
```

Comment:
we do bolleans as such
```suggestion
						"local_nfs_users_with_ldap_allowed": {
```